common/protos/common/events.proto

```diff
@@ -29,21 +43,23 @@ message EventSubscriptionState {
     PENDING = 1;
```

Comment:
Could be `SUBSCRIBE_PENDING` and `SUBSCRIBED` instead of `PENDING` and `SUCCESS` respectively.

Reply:
Let's add SUBSCRIBE_PENDING_ACK and UNSUBSCRIBE_PENDING_ACK as well (and remote PENDING_ACK).